### PR TITLE
Added styling, carousel functionality, basic modal

### PR DESCRIPTION
**Pull request by ericjlie**
@vmichaelwu @yulanrong  A lotta stuff here but please review

### CODE REVIEW

**Comment by yulanrong** [comment]
Hi Eric, your pull request has a merge conflict, can you take a look at it?

File: client/src/component/qa/Question.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const Question = ({ question }) => {
     event.preventDefault();
     console.log(question.question_id);
```
Comment (yulanrong) [inline client/src/component/qa/Question.jsx:37]: can you remove this console.log?

File: client/src/component/relateditems/RelatedItemsMenu.jsx [hunk 2]
```diff
@@ -1,45 +1,110 @@
...
+        changeCurrentRange([upperBound - 5, upperBound]);
+      }
+    } else {
+      changeCurrentRange([currentRange[0] - 1, currentRange[1] - 1]);
+    }
+    changeToShow(relatedList.slice(currentRange[0], currentRange[1]));
+  };
+
   useEffect(() => {
     console.log(productId);
```
Comment (yulanrong) [inline client/src/component/relateditems/RelatedItemsMenu.jsx]: can you remove console.log?

File: server/controllers/qa.js [hunk 3]
```diff
@@ -36,12 +36,13 @@ module.exports = {
     },
 
     helpful: (req, res) => {
+      console.log(req.query.question_id);
```
Comment (yulanrong) [inline server/controllers/qa.js]: can you remove console log?

File: server/controllers/related.js [hunk 4]
```diff
@@ -30,10 +30,10 @@ module.exports = {
           //Calculate average rating by reducing ratings and then dividing,
```
Comment (yulanrong) [inline server/controllers/related.js]: can you remove the comments?

**Comment by ericjlie** [comment]
@yulanrong @vmichaelwu Issues and merge conflict resolved, should be good to merge.

**Pull request merged**